mcf-browser/src/back-end/graph.js

```diff
@@ -13,126 +13,41 @@
- */
-function getLabelsTargetUrl(dcid) {
-  const targetUrl = (API_ROOT + '/node/property-labels?dcids=' + dcid);
-  return targetUrl;
-}
-
-/**
- * Generates the url to get property values from the DC KG using REST API.
- *
- * @param {string} dcid The dcid of the node to get the property values for.
- * @param {string} label The property label to query for.
- * @param {boolean} out The direction of label, true indicates outgoing label.
- * @return {string} The generated url.
- */
-function getValuesTargetUrl(dcid, label, out) {
-  let targetUrl = (API_ROOT + '/node/property-values?dcids=' + dcid +
-                   '&property=' + label + '&direction=');
-  if (out) {
-    targetUrl += 'out';
-  } else {
-    targetUrl += 'in';
-  }
-  targetUrl += '&limit=500';
-  return targetUrl;
-}
-
-/**
- * Gets all property labels of the given dcid that are in the DC KG.
- *
- * @param {string} dcid The dcid of the node to find property labels for.
- * @return {Object} An object containing both 'in' and 'out' property labels.
- */
-async function getRemotePropertyLabels(dcid) {
-  const targetUrl = getLabelsTargetUrl(dcid);
-  return fetch(targetUrl)
-      .then((res) => res.json())
-      .then((data) => JSON.parse(data.payload)[dcid]);
-}
-
 /**
- * Gets all property values containing the given dcid, property label, and
- * direction.
- *
- * @param {string} dcid The dcid of the node to find property value for.
- * @param {string} label The property label to query for.
- * @param {boolean} out The direction of label, true indicates outgoing label.
- * @return {Object} An object containing all found values matching the query.
+ * Module contains Node and Assertion classes which together create a local
+ * version of the Data Commons Knowledge Graph.
  */
-async function getRemotePropertyValues(dcid, label, out) {
-  const targetUrl = getValuesTargetUrl(dcid, label, out);
-  return fetch(targetUrl)
-      .then((res) => res.json())
-      .then((data) => JSON.parse(data.payload)[dcid])
-      .then((triples) => {
-        console.log('url: ' + targetUrl + ', values: ' + triples);
-        if (out) {
-          return triples.out;
-        } else {
-          return triples.in;
-        }
-      });
-}
 
-/**
- * Parses an Object returned from the DC REST get_values API to create a Node
- * object from the value's dcid or to return the string value that the object
- * holds.
- *
- * @param {Object} valueObj An object returned from DC REST get_values API.
- * @return {Node | string} The created Node if the value obj has a dcid,
- *     otherwise the string of the value.
- */
-function getValueFromValueObj(valueObj) {
-  let value;
-  if ('dcid' in valueObj) {
-    value = Node.getNode(valueObj.dcid, true, true);
-    value.existsInKG = true;
-  } else if ('value' in valueObj) {
-    value = valueObj.value;
-  } else {
-    console.log('ERROR remote fetch no dcid or value prop: ' + valueObj);
-  }
-  return value;
-}
+import * as utils from './utils.js'
```

Comment:
It's best to import only what you need from the class (so it's easy to trace the definitions as one reads the code as well).